The next problem when I try to upgrade to a newer persistent version is this error:

```
/home/jakob/projects/yesod-scaffold/test/TestImport.hs:60:48: error:
    • Couldn't match expected type ‘Text’
                  with actual type ‘Database.Persist.SqlBackend.Internal.SqlBackend’
    • In the first argument of ‘getEscapedRawName’, namely ‘sqlBackend’
      In the first argument of ‘map’, namely
        ‘(getEscapedRawName sqlBackend)’
      In the expression: map (getEscapedRawName sqlBackend) tables
   |        
60 |     let escapedTables = map (getEscapedRawName sqlBackend) tables
   |                                                ^^^^^^^^^^
```

I can't find `ask` in the ChangeLog nor in Hoogle. How can I find out where it comes from and how to replace it? Oh, I finally found it in [classy-prelude](https://www.stackage.org/haddock/lts-19.29/classy-prelude-1.5.0.2/ClassyPrelude.html#v:ask). But I'm afraid I don't understand the problem.

--- a/package.yaml
+++ b/package.yaml
@@ -8,31 +8,31 @@ dependencies:
 - yesod-core >=1.6 && <1.7
 - yesod-auth >=1.6 && <1.7
 - yesod-static >=1.6 && <1.7
-- yesod-form >=1.6 && <1.7
+- yesod-form >=1.6 && <1.8
 - classy-prelude >=1.5 && <1.6
 - classy-prelude-conduit >=1.5 && <1.6
 - classy-prelude-yesod >=1.5 && <1.6
 - bytestring >=0.10 && <0.11
 - text >=0.11 && <2.0
-- persistent >=2.9 && <2.11
-- persistent-postgresql >=2.9 && <2.11
-- persistent-template >=2.5 && <2.9
+- persistent >=2.9 && <2.14
+- persistent-postgresql >=2.13 && <2.14
+- persistent-template >=2.5 && <2.14
 - template-haskell
 - shakespeare >=2.0 && <2.1
 - hjsmin >=0.1 && <0.3
 - monad-control >=0.3 && <1.1
-- wai-extra >=3.0 && <3.1
+- wai-extra >=3.0 && <3.2
 - yaml >=0.11 && <0.12
 - http-client-tls >=0.3 && <0.4
 - http-conduit >=2.3 && <2.4
 - directory >=1.1 && <1.4
 - warp >=3.0 && <3.4
 - data-default
-- aeson >=1.4 && <1.5
+- aeson >=1.4 && <2.1
 - conduit >=1.0 && <2.0
 - monad-logger >=0.3 && <0.4
-- fast-logger >=2.2 && <3.1
-- wai-logger >=2.2 && <2.4
+- fast-logger >=2.2 && <3.2
+- wai-logger >=2.2 && <2.5
 - file-embed
 - safe
 - unordered-containers

--- a/src/Model.hs
+++ b/src/Model.hs
@@ -7,9 +7,10 @@
 {-# LANGUAGE OverloadedStrings          #-}
 {-# LANGUAGE TemplateHaskell            #-}
 {-# LANGUAGE TypeFamilies               #-}
-{-# LANGUAGE DerivingStrategies #-}
-{-# LANGUAGE StandaloneDeriving #-}
-{-# LANGUAGE UndecidableInstances #-}
+{-# LANGUAGE DerivingStrategies         #-}
+{-# LANGUAGE StandaloneDeriving         #-}
+{-# LANGUAGE UndecidableInstances       #-}
+{-# LANGUAGE DataKinds                  #-}
 module Model where
 
 import ClassyPrelude.Yesod

--- a/test/TestImport.hs
+++ b/test/TestImport.hs
@@ -57,7 +57,7 @@ wipeDB app = runDBWithApp app $ do
     -- Should your application grow to hundreds of tables and tests,
     -- switching to DELETE could be a substantial speedup.
     -- See: https://github.com/yesodweb/yesod-scaffold/issues/201
-    let escapedTables = map (getEscapedRawName sqlBackend) tables
+    let escapedTables = map (\t -> getEscapedRawName t sqlBackend) tables
         query = "TRUNCATE TABLE " ++ intercalate ", " escapedTables
     rawExecute query []
 

